packages/atomic/src/utils/field-utils.ts

```diff
@@ -0,0 +1,19 @@
+import {i18n} from 'i18next';
+
+function getFieldCaptionNamespace(field: string) {
```

Comment:
Used the field instead of the facet id, I thought it could be more explicitly reused for result templates this way.
It there a case where 2 facets with the same field would have 2 different sets of captions or is it overkill?

Reply:
I can only think of extremely weird cases. I think using the field is perfectly fine.